README.rst

```diff
@@ -25,14 +25,47 @@ Connexion
    :target: https://github.com/zalando/connexion/blob/master/LICENSE
    :alt: License
 
-Connexion is a framework on top of Flask_ to automagically handle your REST API requests
-based on `OpenAPI 2.0 Specification`_ (formerly known as Swagger Spec) files
-in YAML.
+Connexion is a Flask_ application/framework that automagically handles your REST API requests based on `OpenAPI 2.0 Specification`_ (formerly known as Swagger Spec) files in YAML. Written in Python, Connexion enables you to write your Swagger specification, then maps the specification to Python functions. This is what makes it unique from other tools. We built Connexion this way in order to:
+- simplify the development process
+- reduces misinterpretation about what an API is going to look like
+- save time
 
-How to use
+Connexion Features:
 ----------
+- validates requests and endoint parameters automatically, based on your specification  
+- provides the Swagger Console UI so that you can call your API through it
+- handles OAuth 2
+- supports API versioning
+- supports automatic serializization. If your specification defines that an endpoint returns JSON, Connexion will automatically serialize the return value for you and set the right content type in the HTTP header.
 
-Put your API YAML inside a folder in the root path of your application (e.g ``swagger/``) and then do
+Why Connexion
+----------
+With Connexion, you write the spec first. It then calls your Python code—handling the mapping from the specification to the code. This incentivizes you to write the specification so that all of your developers can understand what your API does—even before you write a single line of code. If multiple teams depend on your APIs, you can already send them documentation of your API. This guarantees that your API will follow the specification that you wrote.  
+it’s not like Hug_, which generates the specification based on what code you write. 
+
+Other Sources/Mentions
+----------
+- Zalando Tech blog post `API First`_
+- We're honored that Connexion is Swagger_'s official framework of choice for Python.
```

Comment:
I think this is a bit misleading. Connexion is one of the Python frameworks listed in Swagger website. But it is not the official framework of choice.